web-app/app/modules/messages_module.rb

```diff
@@ -0,0 +1,22 @@
+module MessagesModule
+    
+    def self.send_SMS_message(number,message)
+        
+        require 'twilio-ruby'
+        @client = twilio_connection
+        @message = @client.create({:from => number_cel_from, :to => number, :body => message})
+    end 
+
+    def self.number_cel_from=(number)
+        @@from = number
+    end
+
+    private 
+    def twilio_connection
+        Twilio::REST::Client.new(ENV['TWILIO_ACCOUNT_SID'], ENV['TWILIO_AUTH_TOKEN']).account.sms.messages
+    end
+
+    def self.number_cel_from
+        @@from ||= ENV['NUMBER_CEL_FROM']
+    end
+end
```

Comment:
Can you fix this file's indentation to be 2 spaces wide? (Using spaces, not tabs).
Also, make sure to keep an empty line at the end of the file here and in the rest of the project (so github doesn't display that symbol at the end).